_docker/drupal/drupal-filesystem/web/config/sync/bootstrap.settings.yml

```diff
@@ -5,7 +5,7 @@ _core:
 favicon:
   mimetype: image/vnd.microsoft.icon
   path: null
-  url: ''
+  url: 'https://localhost/themes/contrib/bootstrap/favicon.ico'
```

Comment:
@jordanpagewhite Can we make this relative? `https://localhost` isn't going to work in any of the other environments.

cheers,

Rob

Reply:
Thanks Rob for noticing this. This must have been exported as 'default' config in the latest version of Bootstrap and I missed it. The favicon within the admin section (pages that load the admin theme(s)) remains the same. I've removed this config value either way though just because it seems unnecessary.